config/locales/found.fr.yml

```diff
@@ -1,5 +1,7 @@
 ---
 fr:
+  #more: "More..."
```

Comment:
This comment may be removed

Reply:
For now, I leave all the english strings in the french one, it's easier for me to start from this one to keep the english file up to date. But I agree, all those comments will vanish when the strings will be dispatched correctly.